src/content/docs/apm/apm-ui-pages/events/change-tracking/deployments-page-view-impact-your-app-users.mdx

```diff
@@ -0,0 +1,191 @@
+* All the data supplied when the deployment was recorded, which helps anyone quickly understand context—regardless of whether they made the change or not.
+    <Callout variant="tip">
+      When you record a deployment, try submitting a URL for the changelog attribute. The UI will render it as a clickable link that serves as a convenient jumping-off point to your source repo management system.
+    </Callout>
+
+### Change the before/after comparison period [#change-window]
+
+The **Deployment details** page hinges on the idea that records and signals generated over a period of time leading up to this deployment are being compared with a period of equal length following this deployment. You can change the length of that period using a time picker in the top-right corner. Note that this will affect both time series charts and other UI elements featuring data-driven insights.
+
+<img
+  style={{ width: "25%" }}
+  title="A screenshot showing how to change the time window for the comparison"
+  alt="A screenshot showing how to change the time window for the comparison"
+  src={trackingComparisonWindow}
+/>
+
+<Callout variant="tip">
+  Remember that the relevance of net impact and average values depends on the timepicker selection. For example, if a deployment happened 30 minutes ago and you are comparing the hour before it to the hour after it, the relevance of some insights may be a little unbaked, so to speak, and in some cases the UI will not even attempt to display the result.
+</Callout>
+
+### Compare one deployment to another [#compare-deployments]
+
+Next to the timepicker in the top-right corner of the **Deployment details** page, you will see **compared with** next to a dropdown menu. Using that dropdown menu, you can select another recorded deployment. This will toggle the **Deployment marker details** page to a comparative mode.
+
+When you compare deployments:
+
+* Each time series chart will show a curve for THIS deployment and THAT deployment, which will help you compare the relative performance of each deployment.
+* Other calculations on this page will update to help you understand the difference between THIS deployment and THAT deployment. They may show increases/decreases in percentages for rates or counts (for example, +11) where it makes sense to do so.
+* You can open up to five stacked views of related deployments.
+
+## New UI: Query deployments data [#query-deployments]
+
+You can also query deployments data via NerdGraph (New Relic's GraphQL API) or NRQL (the query language for the New Relic database). For more details, see [Introduction to New Relic NerdGraph](/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph/) and [Introduction to NRQL](docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language/).
```

Comment:
I've read in our style guide that is better to avoid the use of apostrophes with NR. See [Trademark guidelines](https://docs.newrelic.com/docs/style-guide/capitalization/product-capability-feature-usage).

Reply:
@nbaenam good catch! I just pushed up a fix for this.